go/billing/models.py

```diff
@@ -216,10 +216,22 @@ class Transaction(models.Model):
         (STATUS_REVERSED, STATUS_REVERSED),
     )
 
+    TRANSACTION_TYPE_MESSAGE = 'Message'
+    TRANSACTION_TYPE_TOPUP = 'Top Up'
+    TRANSACTION_TYPE_CHOICES = (
+        (TRANSACTION_TYPE_TOPUP, TRANSACTION_TYPE_TOPUP),
+        (TRANSACTION_TYPE_MESSAGE, TRANSACTION_TYPE_MESSAGE),
```

Comment:
I originally had it like that, and also prefer it, but changed it to capitals to stay consistent with what we are doing for the status field on `Transaction` and the message direction field on `MessageCost`.